src/cephadm/cephadm.py

```diff
@@ -1035,17 +1035,32 @@ def get_container_binds():
         return binds
 
     @staticmethod
-    def get_version(ctx, container_id):
-        # type: (CephadmContext, str) -> Optional[str]
-        version = None
-        out, err, code = call(ctx,
-                              [ctx.container_engine.path, 'exec', container_id,
-                               '/usr/bin/python3', '-c',
-                               "import pkg_resources; print(pkg_resources.require('ceph_iscsi')[0].version)"],
-                              verbosity=CallVerbosity.QUIET)
+      def python(s: str) -> Tuple[str, str, int]:
+            return call(
```

Comment:
can keep the # type comment

Reply:
I believe that I have made this change correctly.